### PR TITLE
contracts-bedrock: delete dead typescript

### PR DESCRIPTION
**Pull request by tynes**
**Description**

This commit deletes a bunch of dead typescript code from the contracts-bedrock package.

This deletes bundling in all of the artifacts into the npm package so it will make the npm package must smaller. It will also reduce the build time by removing some steps from the build.

### CODE REVIEW

**Review by refcell — APPROVED** [review]
woot woot 🎉

File: packages/contracts-bedrock/package.json [hunk 1]
```diff
@@ -57,35 +60,19 @@
...
     "ds-test": "https://github.com/dapphub/ds-test.git#9310e879db8ba3ea6d5c6489a579118fd264a3f5",
-    "ethereum-waffle": "^3.0.0",
     "forge-std": "https://github.com/foundry-rs/forge-std.git#e8a047e3f40f13fa37af6fe14e6e06283d9a060e",
     "glob": "^7.1.6",
-    "hardhat": "^2.9.6",
-    "hardhat-deploy": "^0.11.4",
     "solhint": "^3.3.7",
     "solhint-plugin-prettier": "^0.0.5",
     "ts-node": "^10.9.1",
-    "typechain": "^8.1.0",
```
Comment (roninjin10) [inline packages/contracts-bedrock/package.json:88]: All these deps being nuked is amazing

File: packages/sdk/src/adapters/standard-bridge.ts [hunk 2]
```diff
@@ -13,8 +13,10 @@ import {
   BlockTag,
 } from '@ethersproject/abstract-provider'
 import { predeploys } from '@eth-optimism/contracts'
-import { getContractInterface } from '@eth-optimism/contracts-bedrock'
 import { hexStringEquals } from '@eth-optimism/core-utils'
+import l1StandardBridgeArtifact from '@eth-optimism/contracts-bedrock/forge-artifacts/L1StandardBridge.sol/L1StandardBridge.json'
+import l2StandardBridgeArtifact from '@eth-optimism/contracts-bedrock/forge-artifacts/L2StandardBridge.sol/L2StandardBridge.json'
+import optimismMintableERC20 from '@eth-optimism/contracts-bedrock/forge-artifacts/OptimismMintableERC20.sol/OptimismMintableERC20.json'
```
Comment (roninjin10) [inline packages/sdk/src/adapters/standard-bridge.ts:19]: We could clean up these imports if we wanted with [exports](https://github.com/wagmi-dev/wagmi/blob/main/packages/react/package.json#L30) in a future pr

**Review by roninjin10 — APPROVED** [review]
lgtm pending ci

File: packages/sdk/src/adapters/standard-bridge.ts (hunk 2 above)
Comment (tynes) [reply packages/sdk/src/adapters/standard-bridge.ts:19]: Would love to clean this up in a future pr

**Comment by tynes** [comment]
Will rebase on https://github.com/ethereum-optimism/optimism/pull/6152

**Pull request merged**